facebook/OpenBIC
# fby3.5: cl: Added memory thermal trip event.

Summary:
- Added event log for memory thermal trip assert event.

Test Plan:
- Build code: Pass

Log:
1.  Check BIC will send memory thermal trip event when event happened.
[BIC console]
uart:~$ gpio get GPIO0_E_H 26
Reading GPIO0_E_H pin 26
Value 1
uart:~$ gpio conf GPIO0_E_H 26 out
Configuring GPIO0_E_H pin 26
uart:~$ gpio get GPIO0_E_H 26
Reading GPIO0_E_H pin 26
Value 0
uart:~$ gpio get GPIO0_A_D 20
Reading GPIO0_A_D pin 20
Value 1
uart:~$ gpio conf GPIO0_A_D 20 out
Configuring GPIO0_A_D pin 20
uart:~$ gpio get GPIO0_A_D 20
Reading GPIO0_A_D pin 20
Value 0

[BMC console]  // BMC would modify event data to match BIC added event.
root@bmc-oob:~# log-util slot2 --print
SEL Entry: FRU: 2, Record: Standard (0x02), Time: 2022-02-17 16:32:41, Sensor: SYSTEM_STATUS (0x10), Event Data: (11FFFF) Slot255 present Assertion

2.  Check BIC will send SOC thermal trip event when event happened.
[BIC console]
uart:~$ gpio get GPIO0_E_H 26
Reading GPIO0_E_H pin 26
Value 1
uart:~$ gpio get GPIO0_A_D 20
Reading GPIO0_A_D pin 20
Value 1
uart:~$ gpio conf GPIO0_A_D 20 out
Configuring GPIO0_A_D pin 20
uart:~$ gpio get GPIO0_A_D 20
Reading GPIO0_A_D pin 20
Value 0

[BMC console]
root@bmc-oob:~# log-util slot2 --print
SEL Entry: FRU: 2, Record: Standard (0x02), Time: 2022-02-17 16:33:37, Sensor: SYSTEM_STATUS (0x10), Event Data: (00FFFF) SOC Thermal trip Assertion
